udemy_exercises/songs/Song.java

```diff
@@ -1,13 +1,18 @@
-package udemy_exercises.songs;
+public class Song {
+    private String title;
```

Comment:
The data type 'String' accepts the value [null].  How might this affect the methods of your implementation?